recipes/flipnslide/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "flipnslide" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d65e7c220d43eb55da8f7bccdd518779a53a5802b16967054bba2bef8b035405
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - flipnslide
+  commands:
+    - pip check
+  requires:
+    - pip
+    - numpy
+    - torch
+    - pandas
+    - numpy
+    - matplotlib
```

Comment:
You likely only need `matplotlib-base`, which doesn't include all of the qt packages.